### PR TITLE
[HOPSWORKS-2960] User management 

### PR DESCRIPTION
**Pull request by ErmiasG**
## Make sure there is no duplicate PR for this issue

* **Please check if the PR meets the following requirements**
- [ ] Adds tests for the submitted changes (for bug fixes & features)
- [ ] Passes the tests
- [x] HOPSWORKS JIRA issue has been opened for this PR
- [ ] All commits have been squashed down to a single commit

* **Post a link to the associated JIRA issue**

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: docs/admin/user.md [hunk 1]
```diff
@@ -1 +1,123 @@
...
+<figure>
+  <a  href="../../assets/images/admin/user-management/new-user.png">
+    <img src="../../assets/images/admin/user-management/new-user.png" alt="New user" />
+  </a>
+  <figcaption>Create new user</figcaption>
+</figure>
+
+After setting the user's name and email chose the type of user you want to create (Hopsworks, Kerberos or LDAP). To 
+create a Kerberos or LDAP user you need to get the users **UUID** from the Kerberos or LDAP server. Hopsworks user 
+can also be assigned a _Role_. Kerberos and LDAP users on the other hand can only be assigned role through group
```
Comment (robzor92) [inline docs/admin/user.md:76]:
```suggestion
can also be assigned a _Role_. Kerberos and LDAP users on the other hand can only be assigned a role through group 
```

**Review by robzor92 — CHANGES_REQUESTED** [review]

File: docs/admin/user.md [hunk 2]
```diff
@@ -1 +1,123 @@
...
+  </a>
+  <figcaption>Active Users</figcaption>
+</figure>
+
+### Cluster roles
+
+Roles let you manage the access rights of a user to the cluster.
+
+- User: users with this role are only allowed to use the cluster by creating a limited number of projects.
+- Admin: users with this role are allowed to manage the cluster. This includes accepting new users to the cluster or
```
[the remark this answers is not in the document]
Comment (ErmiasG) [reply docs/admin/user.md:23]: They can manage quota, so they can change project limit

File: docs/admin/user.md [hunk 3]
```diff
@@ -1 +1,123 @@
...
+  blocking them, managing user quota, [configure alerts](./alert.md) and setting up [authentication methods](./auth.md).  
+
+## Validating and blocking users
+By default, user who register on Hopsworks using their own credentials are not granted access to the cluster. 
+First, a user with an admin role needs to validate their account.
+
+By clicking on the _Review Requests_ button you can open a _user request review_ popup as shown in the image below.
+
+<figure>
+  <a  href="../../assets/images/admin/user-management/user-request-review.png">
```
[the remark this answers is not in the document]
Comment (ErmiasG) [reply docs/admin/user.md:33]: It is dummy data

File: docs/admin/user.md [hunk 4]
```diff
@@ -1 +1,123 @@
...
+    <img src="../../assets/images/admin/user-management/active-users.png" alt="active users" />
+  </a>
+  <figcaption>Active Users</figcaption>
+</figure>
+
+### Cluster roles
+
+Roles let you manage the access rights of a user to the cluster.
+
+- User: users with this role are only allowed to use the cluster by creating a limited number of projects.
```
[the remark this answers is not in the document]
Comment (ErmiasG) [reply docs/admin/user.md:22]: No, it needs to be added to the design.

**Pull request merged**